lighthouse-cli/sentry-prompt.js

```diff
@@ -29,29 +29,25 @@ function prompt() {
   /** @type {NodeJS.Timer|undefined} */
   let timeout;
 
-  const prompt = inquirer.prompt([
-    {
-      type: 'confirm',
-      name: 'isErrorReportingEnabled',
-      default: false,
-      message: MESSAGE,
-    },
-  ]);
+  const prompt = new Confirm({
+    name: 'isErrorReportingEnabled',
+    initial: false,
+    message: MESSAGE,
+  });
```

Comment:
to take care of enquirer preventing `ctrl-c` from working, I think the best approach (based on https://github.com/GoogleChrome/lighthouse/issues/6658#issuecomment-442887845 and a quick debug session...there may be a better way but I don't see it) is adding

```suggestion
    actions: {ctrl: {}},
  });
```
(and `types/enquirer/index.d.ts` will also have to be updated)

This gets rid of all the `ctrl-*` key handling, but since we're only using a y/N prompt, I don't think we're going to miss any of it (e.g. if it was text entry we'd probably want to keep `ctrl-a` and `ctrl-e`, but we don't have that).

It's still a bit weird. `ctrl-c` during the prompt still cancels the whole run, but answering yes or no and then doing `ctrl-c` during the lighthouse run kills the lighthouse process but not the Chrome one. That's less than ideal, but it's better than `ctrl-c` not doing anything, and this is only on the first run when the user is prompted.

Reply:
I added `actions` to the option. Also, I updated the path of .d.ts as suggested at https://github.com/GoogleChrome/lighthouse/issues/12305#issuecomment-810409143 (from `types/enquirer/index.d.ts` to `types/enquirer.d.ts`).